js/init_and_modules_test.go

```diff
@@ -28,16 +28,16 @@ import (
 	"testing"
 	"time"
 
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 	"go.k6.io/k6/js"
 	"go.k6.io/k6/js/common"
 	"go.k6.io/k6/js/modules"
 	"go.k6.io/k6/lib"
 	"go.k6.io/k6/lib/testutils"
 	"go.k6.io/k6/loader"
 	"go.k6.io/k6/stats"
-	"github.com/spf13/afero"
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
 	"gopkg.in/guregu/null.v3"
```

Comment:
no I haven't ... and I am against it as the whole idea of gofmt (and goimports respectively) is for them to work the same for everybody and to preferably have 0 knobs ... 

Even if I decide to go through the trouble of doing this is everyone who contributes will likely not do it so we will need to ask them to ... fix it, which IMO is a waste of everybodies time, as again you probably shouldn't go look at 20 lines of imports and care how they are ordered. 

Don't get me wrong if goimprots decides to default to first putting the names of the module you are in :+1: , but me telling it that it should do it is a totally different thing.

Reply:
> I am against it as the whole idea of gofmt (and goimports respectively) is for them to work the same for everybody and to preferably have 0 knobs

Yeah, but they don't :smile:

I don't think we should insist on this for external contributors, but we at least can have that configured so that whenever we change a file it's updated. It's a minor thing, but increases readability of imports substantially IMO.

Reply:
> I don't think we should insist on this for external contributors, but we at least can have that configured so that whenever we change a file it's updated. 

And apparently, if it's not configured and someone contributes it will move them ... so this IMO should just not be a thing